include/picongpu/plugins/radiation/radFormFactor.hpp

```diff
@@ -142,15 +142,43 @@ namespace picongpu
                         /* currently a fixed sigma of DELTA_T * c is used to describe the distribution - might become a
                          * parameter */
                         return math::sqrt(
-                            N
-                            + (N * N - N)
-                                * util::square(
-                                    math::exp(float_X(-0.5) * util::square(omega * float_X(0.5) * DELTA_T))));
+                            N + (N * N - N) * math::exp(float_X(-1.0) * util::square(omega * float_X(0.5) * DELTA_T)));
                     }
                 };
             } // namespace radFormFactor_Gauss_spherical
 
 
+            namespace radFormFactor_Gauss_spherical_simple
+            {
+                struct radFormFactor
+                {
+                    /** Form Factor for point-symmetric Gauss-shaped charge distribution of N discrete electrons:
+                     * \f[ <rho(r)> = N*q_e* 1/sqrt(2*pi*sigma^2) * exp(-0.5 * r^2/sigma^2) \f]
+                     * with sigma = 0.5*c/delta_t (0.5 because sigma is defined around center)
+                     * simplified to avoid calling too often special functions to:
+                     * sqrt( N + (N^2 - N) * F(omega) )
+                     * F(omega) = ( exp(-0.5 * (x/2.0 * delta_t)^2) )^2 ~ 1/(alpha*(delta_t)^2 * x^2 + 1)^beta
+                     *
+                     * @param N = macro particle weighting
+                     * @param omega = frequency at which to calculate the  form factor
+                     * @param observer_unit_vec = observation direction
+                     * @return the Form Factor: \f$ \sqrt( | \mathcal{F} |^2 ) \f$
+                     */
+                    HDINLINE float_X
+                    operator()(const float_X N, const float_X omega, const vector_X observer_unit_vec) const
+                    {
+                        /* currently a fixed sigma of DELTA_T * c is used to describe the distribution - might become a
+                         * parameter */
+                        // optimized paramter for exponent beta=16 (see picongpu PR #3696 for details):
+                        const float_X alpha = 0.0172169f;
```

Comment:
Can be `constexpr` ?

Reply:
Also for `float_X` the suffix is `_X`, not `f` (which is for `float`).

Reply:
Yes as @sbastrakov said please use `0.0172169_X` to be able to switch PIConGPU's precision to 64bit without float upscale.